src/SourceEngine.Demo.Stats.App/Properties/launchSettings.json

```diff
@@ -2,7 +2,7 @@
   "profiles": {
     "SourceEngine.Demo.Stats.App": {
       "commandName": "Project",
-      "commandLineArgs": "IDemO -folders \"demos\" -output \"parsed\" -recursive -samefilename -samefolderstructure -nochickens"
+      "commandLineArgs": "IDemO -folders \"demos\" -output \"parsed\" -recursive -samefilename -samefolderstructure -nochickens -gamemodeoverride \"dangerzone\" -testtype \"competitive\""
```

Comment:
This change probably shouldn't be checked in. It could be a separate profile, but I never use these, so I don't even remember how they work.

Reply:
Yeah these are new optional parameters so I shouldn't have committed them, made their way in due to testing, cheers